api/controllers/job.py

```diff
@@ -33,30 +34,33 @@ def get_jobs_on_home_page(query: s.JobHomePage, current_user: m.User, db: Sessio
         sa.select(m.Job).join(m.SavedJob).where(m.SavedJob.user_id == current_user.id)
     ).all()
 
+    recommended_jobs_out = [
+        s.JobCard(
+            location=s.LocationStrings(
+                uuid=job.location.uuid,
+                name=job.location.region[0].name_ua if query.lang == CFG.UA else job.location.region[0].name_en,
+            ),
+            is_saved=job in user_saved_jobs,
+            **pop_keys(job.__dict__, ["location"]),
```

Comment:
This code looks messy. I think in can be reafactored using `@property` decorator in job model